diff -r 980430f18066 tools/import/import.pro
--- a/tools/import/import.pro	Mon Feb 22 12:26:54 2010 +1100
+++ b/tools/import/import.pro	Sun Mar 07 20:11:19 2010 +0100
@@ -45,10 +45,9 @@
 !macx {
 
 INCLUDEPATH += . \
-	../../3rdParty/include \
-        ../../3rdParty/include/ImageMagick
+        /usr/include/ImageMagick
 
-LIBPATH += ../../3rdParty/lib
+LIBPATH += /usr/lib
 
 LIBS += -lnetcdf_c++ \
         -lnetcdf \
